lib/gateway/pipelines.js

```diff
@@ -140,40 +140,41 @@ function configurePipeline (pipelinePoliciesConfig, config) {
   validatePipelinePolicies(pipelinePoliciesConfig, config);
 
   pipelinePoliciesConfig.forEach(policyConfig => {
-    let policyName = Object.keys(policyConfig)[0];
-    let policySteps = policyConfig[policyName];
+    if (policyConfig && typeof policyConfig === 'object') {
```

Comment:
I suggest to negate if and return instead of putting entire code inside block

Reply:
and if there is no policies in pipeline do not enter forEach at all, not sure if the line is needed at all. 
if configuration is invalid it should at least fail (current behavior), preferably with useful description